src/common/common.css

```diff
@@ -1,24 +1,49 @@
+/* Photon Colors CSS Variables v3.2.0
+   https://firefoxux.github.io/design-tokens/photon-colors/photon-colors.css */
+:root {
+  --blue-50: #0a84ff;
+  --blue-50-a30: rgba(10, 132, 255, 0.3);
```

Comment:
> Since the base color was defined in hex I could not use rgba to generate the error background witha transparency, 

So as you modify the copy here anyway, why not convert the orginal colors into rgb format? (But would it actually work anyway?) As you modified the code after coping from the source anyway, one cannot easily copy & paste it anyway?

Reply:
Yes If I go ahead and change the base color to rgb format it would work.  

It is just to maintain consistency within code, we have to convert the base hex code into rgb and then use it. I will go ahead and convert the base color into rgb.